tests/wps_restapi/test_jobs.py

```diff
@@ -1579,6 +1586,26 @@ def test_jobs_inputs_outputs_validations(self):
         with self.assertRaises(colander.Invalid):
             sd.Execute().deserialize({"outputs": {"random": {"transmissionMode": "bad"}}})
 
+    def test_job_transforms(self):
+        # First try transformation engine
+        test_transformations()
+
+        # Then try transformation in action
+        path = f"/jobs/{self.job_info[0].id}/outputs"
+        resp = self.app.get(path, headers=self.json_headers)
+        for link in resp.json["links"]:
+            header = {"Accept": link["type"], "Content-Type": link["type"]}
```

Comment:
The `"Accept": link["type"]` look valid, but `"Content-Type": link["type"]` doesn't make sense. 
You should remove it because the GET requests do not have a body, and therefore no `Content-Type` header applies.